### DISCUSSION TITLE
How y'all pronounce SeaXNG?

### DISCUSSION THREAD

**Discussion by lemonadeforlife** [searxng/searxng#2522] [General]
I know I might sound stupid but it's important for translation.

**Comment by Danie10** [comment]
I was wondering the same thing. I've heard seer-x-ng as well as "Search-X-NG". The "NG" is pronounced like the "ng" in "sing".

**Comment by virtadpt** [comment]
"Serks-enn-gee."

**Comment by return42** [comment]
In searX the X stands for the Greek letter [Chi](https://en.wikipedia.org/wiki/Chi_(letter)) / similar to how it is used in [TeX](https://en.wikipedia.org/wiki/TeX) .. SearXNG was a fork of searX and the  NG stands for *"next generation"*. I don't know how to pronounce such a construct; 

#### *"the unpronounceable search engine aka SXNG"*

**Reply by Danie10** [reply]
Thanks that seems to agree then more or less with what @virtadpt stated above then for the X. I'll see if SearX is pronounced anywhere too, as it just has the NG appended on the end really.

**Reply by TRusselo** [reply]
> In searX the X stands for the Greek letter [Chi](https://en.wikipedia.org/wiki/Chi_(letter)) / similar to how it is used in [TeX](https://en.wikipedia.org/wiki/TeX) .. SearXNG was a fork of searX and the NG stands for _"next generation"_. I don't know how to pronounce such a construct;
> 
> #### _"the unpronounceable search engine aka SXNG"_

NG has a pronunciation if it is preceeded by an 'i'. look at every word that ends in "ing"
if only the app name had an 'i' before the NG, oh wait it does

Sear X NG 
x=chi
Sear(chi)ng
Searching

It was named perfectly with Next Gen addition.
"searching"  ends up a valid pronunciation.
pfft. to the "seer NG" folk. they are as bad as the ones that say "gif" with a J sound.

**Comment by meonkeys** [comment]
on https://selfhosted.show/133 they hypothesize it is pronounced like the English word "searching" ("sir" + "ching")

**Reply by Danie10** [reply]
Also makes sense! The plot thickens. This is probably a project that really could do with a "How do you pronounce it" section in the readme...

**Comment by Danie10** [comment]
Seeing the project was forked from SearX as a NG alternative, it makes sense to look at the pronunciation of SearX with NG appended to it. Wikipedia has SearX pronounced as 'serks' (see https://en.wikipedia.org/wiki/Searx) so SearXNG could then logically be 'serks-enn-gee'  or 'serks-NG'. This is what @virtadpt already proposed above as well - I'm just fleshing it out.
